schema/json.go

```diff
@@ -51,12 +51,20 @@ func (dst *JSON) Set(src interface{}) error {
 
 	switch value := src.(type) {
 	case string:
-		*dst = JSON{Bytes: []byte(value), Status: Present}
+		if value == "" {
```

Comment:
Can you test with `*string` and `string` cases removed from the `switch`? Just let it fall to `default`?

Reply:
If we do that, then all of those strings will just be turned into `{}` rather than `""`... Depends on the behavior we want...